tox.ini

```
@@ -168,3 +168,11 @@ passenv =
 skip_install = false
 allowlist_externals =
   git
+
+[testenv:changelog-draft]
+deps =
+    towncrier
+    -r{toxinidir}/docs/requirements.txt
+commands =
+    towncrier --version
```

We should probably separate these.
```suggestion
commands_pre =
    towncrier --version
commands =
```